src/Avalonia.Controls/TransitioningContentControl.cs

```diff
@@ -55,6 +63,15 @@ public bool IsTransitionReversed
         set => SetValue(IsTransitionReversedProperty, value);
     }
 
+  /// <summary>
+  /// Contains the previously assigned <see cref="ContentControl.Content"/>'s property value after it has been replaced (transitioned to) the new content, and so that this "old" content is no longer needed. This property can be bound to by the ViewModel to safely dispose of or otherwise deal with the old content.
+  /// </summary>
+  public object? OldContent {
+    get => _OldContent;
+    private set => SetAndRaise(OldContentProperty, ref _OldContent, value);
+  }
+  object? _OldContent;
```

Comment:
Naming: should be `_oldContent`

Reply:
Thank you for the remarks about style. Will fix it. Not sure what was wrong with indent, I guess GitHub does not handle it too well when editing.

Reply:
And about i being an event. Perhaps - I am still relatively new to Avalonia and not too familiar on how to implement events. So if you can suggest a concrete edit, it would be nice.